build/toolchain/linux/BUILD.gn

```diff
@@ -80,6 +80,7 @@ gcc_toolchain("clang_x64") {
   nm = "nm"
   ar = "ar"
   ld = cxx
+  strip = "strip"
```

Comment:
you can really simplify the android build too if you use this mechanism instead of what's here:

https://github.com/flutter/engine/blob/master/build/toolchain/android/BUILD.gn#L62

but that might be better as a separate patch.